boards/pygamer/Cargo.toml

```diff
@@ -65,15 +57,3 @@ chip = "ATSAMD51J19A"
 [[example]]
 name = "usb_poll"
 required-features = ["usb"]
-
-[[example]]
-name = "sd_card"
```

Comment:
Did you remove these examples from the manifest because they no longer have required features?

I think as a bear practice we should declare all examples in Cargo.toml, even those which don't require extra features.

Reply:
Previously, only examples that required features where listed in `Cargo.toml`. Per the above, several features are no longer used in the actual library (the dependencies were simply moved to the dev dependencies) so were removed. The examples that needed those features were removed from the manifest. If we want to declare them all in the manifest as best practice, I certainly have no problem adding them back in.

Reply:
Yes, let's declare them all in the manifest. Thanks!

Reply:
Completed.